xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1781,6 +1781,8 @@ void CVideoPlayer::ProcessRadioRDSData(CDemuxStream* pStream, DemuxPacket* pPack
 
 bool CVideoPlayer::GetCachingTimes(double& level, double& delay, double& offset)
 {
+  static bool cache_low_level = true;
```

Comment:
no sure but those values might be worth tracking with: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/Process/ProcessInfo.h